linera-chain/src/data_types.rs

```diff
@@ -633,7 +643,7 @@ impl Event {
         use MessageKind::*;
         match self.kind {
             Protected | Tracked => false,
-            Simple | Bouncing => true,
+            Simple | Bouncing => self.grant == Amount::ZERO,
```

Comment:
If `refund_grant_to.is_none()`, is there a reason to forbid skipping?

Reply:
Well, yes if we follow your other comment :)

Reply:
.. except that it never happens